docs/tooling/cli-guides/create-a-validator.md

```diff
@@ -0,0 +1,102 @@
+# Start a Validator with 1-command
+
+This page will guide you to start your own Avalanche Validator through Avalanche-CLI with only
+1 command!
+
+Currently, we are only supporting Fuji network with Mainnet support coming in the near future!
+
+:::warning
+
+ALPHA WARNING: This command is currently in experimental mode. Proceed at your own risk.
+
+:::
+
+## Prerequisites
+
+Before we begin, you will need to:
+
+- Create an AWS account and have an AWS `credentials` file in home directory, More info can be
+  found [here](https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html#file-format-creds)
+- Install [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+- Install [Ansible](https://adamtheautomator.com/install-ansible/)
+
+Currently, we are only supporting AWS cloud service, but we plan to add more cloud services in the
+near future.
+
+## Starting a Validator
+
+To start an Avalanche validator, run:
+
+```shell
+avalanche node create <clusterName>
+```
+
+The created node will be part of cluster `clusterName` and all avalanche node commands applied to
+cluster `clusterName` will apply to all nodes in the cluster.
+
+:::note
+
+Please note that running a validator on AWS will incur cost.
+
+Ava Labs is not responsible for the cost incurred from starting up Avalanche validator through
+Avalanche-CLI
+
+:::
+
+Currently, we have set the following specs of the AWS cloud server to a fixed value, but we are
+planning to implement customization feature in the near future:
+
+- OS Image: `Ubuntu 20.04 LTS (HVM), SSD Volume Type`
+- Instance Type: `c5.2xlarge`
+- Storage: `1 TB`
+
+The command will first ask which region you want to set up your cloud server in: 
+
+```text
+ Which AWS region do you want to set up your node in?: 
+  ▸ us-east-1
+    us-east-2
+    us-west-1
+    us-west-2
+    Choose custom region (list of regions available at https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html)
+```
+
+Note: There is a limit to how many Elastic IPs an AWS account can have at a region (currently it is 
```

Comment:
An AWS account have a limit of 5 Elastics IPs per region. If you need to create more than 5 cloud servers, you will need to use different AWS regions.